utilities/new_client.py

```diff
@@ -0,0 +1,297 @@
+# Copyright 2018 Google LLC
```

Comment:
Maybe create a requirements.txt file as well so folks know what they need to install to run this?